src/poliastro/core/elements.py

```diff
@@ -11,6 +11,31 @@
 from poliastro.core.util import rotation_matrix
 
 
+@jit
+def eccentricity_vector(k, r, v):
```

Comment:
What about adding a docstring in here? 📝

Reply:
On it! 💪🏽

Reply:
Done!